### PR TITLE
markdown: format markdown files

### PR DESCRIPTION
**Pull request by 1480c1**
Cosmetic changes for some files and functional for others

Some changes are based off https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md

a lot of the changes don't change how the file is viewed through GitHub, just in the editor.

### CODE REVIEW

File: test/Readme.md [hunk 1]
```diff
@@ -1,114 +1,130 @@
...
-  # run all the tests in Unit tests
-  ./SvtAv1UnitTests
-  # list the available test cases:
-  ./SvtAv1UnitTests --gtest_list_tests
-  # or run the specific test cases, for example the test cases whose name contain "transform"
-  ./SvtAv1UnitTests --gtest_filter="*transform*"
-
-  ```
+
+### MacOS or Linux(64-bit)
```
Comment (EwoutH) [inline test/Readme.md]: Space between Linux and (64-bit)

Comment (1480c1) [reply test/Readme.md]:
```suggestion
### MacOS or Linux (64-bit)
```

**Comment by hassount** [comment]
@1480c1 ready to go ?

**Comment by 1480c1** [comment]
Yes

**Pull request merged**